test/Macros/lit.local.cfg

```diff
@@ -25,6 +25,8 @@ else:
         0,
         (
             '%swift-build-cxx-plugin',
```

Comment:
Makes me think `swift-build-cxx-plugin` is the wrong name, since it's not C++ 😅.

Reply:
Yeah that's fine, I just noticed because you added C_FALGS and I was about to comment with "CXX_FLAGS?".